samples/web/asp-web-application/Core_7/WebApp/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Builder;
```

Comment:
thinking a bit more about this, maybe we should leave the v7 version on the "old" generic host model to have both being shown.

Reply:
discussed this and we agreed to keep the samples on the minimal version across the board